extensions/wikia/AdEngine3/src/index.js

```diff
@@ -127,6 +127,17 @@ function trackTabId() {
   pageTracker.trackProp('tab_id', window.tabId);
 }
 
+function trackKruxSegments() {
+	const kruxUserSegments = context.get('targeting.ksg') || [];
+	const kruxICBMSegments = context.get('services.krux.icbmSegments') || [];
```

Comment:
Can we rename it to `kruxTrackedSegments` and `services.krux.trackedSegments`? I don't see the point of using ICBM in the variable names since almost all context values comes from ICBM.